docs/guide/options.md

```diff
@@ -11,7 +11,7 @@ Fonts use the same format as [chart.js](https://www.chartjs.org/docs/master/gene
 ## Scriptable Options
 
 As with most options in chart.js, the annotation plugin options are scriptable. This means that a function can be passed which returns the value as needed. In the example below, the annotation is hidden when the screen is less than 1000px wide.
-The function receives 2 arguments, first is the [option context](#option-context) representing contextual information. An options resolver is passed as second argumet, which can be used to access other option in the same context.
+The function receives 2 arguments, first is the [option context](#option-context) representing contextual information. An options object is passed as second argument, containing the options that were originally passed in to configure the annotation element.
```

Comment:
Its a proxy object though, that resolves from the options through the annotation type defaults to the annotation defaults. But what ever wording makes it more usable to someone who did not implement it, should be used :)

If you access `options.whatEver` from scriptable `options.display` and the `whatEver` is also scriptable, its called with the same context the `display` was called. Another fancy thing is that you can actually resolve literal `whatEver` option, even though its not built-in.

Reply:
Updated. Thank you,